stdlib/jdbc/src/main/java/org/ballerinax/jdbc/Constants.java

```diff
@@ -165,6 +165,9 @@
         public static final String MSSQL_SERVER = "microsoft sql server";
     }
 
+    // Warning suppression
+    public static final String UNCHECKED = "unchecked";
```

Comment:
Why do we need to define this constant like this? This is not a jdbc related constant. @daneshk / @wggihan Are we following similar pattern in other modules?

Reply:
AFAIK, directly using annotation without using this kind of constant in other modules.